src/pages/tug/index.page.js

```diff
@@ -0,0 +1,218 @@
+import styles from './index.module.scss';
+import tugData from '~/data/tug';
+import { CoreLayout } from '~/layouts';
+import { TugHead } from '~/components';
+import Button from '../../components/Button/Button';
+import HomeMVA from '../../components/home/HomeMVA/HomeMVA';
+
+export const getStaticProps = () => {
+  const title = 'TiDB User Group 会员';
+  const description =
+    'TUG(TiDB User Group) 为 TiDB 用户提供了一个开放的交流平台。' +
+    '成员们积极在社区中贡献了大量优质的内容，既扩大了 TUG 的影响力，也从中收获很多价值，形成了“来自社区，回馈社区” 的良性循环。';
+
+  const applyUrl = '/tug/apply';
+
+  const tugOrganizationChartTitle = 'TUG 组织架构';
+  const tugOrganizationChartUrl = '/images/people/tug-organization-chart.svg';
+
+  const howToBecomeTitle = '如何成为 TUG 会员';
+  const howToBecome = [
+    { index: 1, iconUrl: 'images/people/how-to-become-1.svg', title: '填写信息', link: applyUrl },
+    { index: 2, iconUrl: 'images/people/how-to-become-2.svg', title: '信息审核' },
+    { index: 3, iconUrl: 'images/people/how-to-become-3.svg', title: '邀请加入' },
+  ];
+
+  const progressTitle = 'TUG 会员进阶路线';
+  const progressSummary =
+    'TUG 核心成员包括 TUG 管理成员、Active Ambassador、Ambassador。' +
+    '积极活跃的 TUG 成员可被提名为 Ambassador，Ambassador 在社区贡献优质内容可升级为 “Active Ambassador” 之后有两种不同的发展方向。';
+  const progressStep1 =
+    '管理线：上升通道为 TUG 管理成员，包括 Team Co-Leader、Team Leader、TMC 成员。TUG 管理成员任期为一年。' +
+    'TMC 成员、Team Leader 每年通过换届选举产生，Team Co-Leader 在换届选举后由 TeamLeader 进行委任。';
+  const progressStep2 =
+    '专业线：上升通道为 TiDB MVA (Most Valuable Advocate)，或者更高级别的 TiDB MOA (Most OutstandingAdvocate)。';
+
+  const peopleMemberTitle = 'TMC 成员';
+  const peopleMemberSummary = 'TMC 是 TUG 最高决策组织，负责统筹 TUG 发展与规划、重要人事任免、全年大事安排等';
+  const peopleLeaderTitle = 'LEADER';
+  const peopleLeaderSummary = 'TUG 各区域负责人，负责统筹本区域内TUG 的发展以及活动规划';
+
+  const { tmc, leader } = tugData;
+
+  return {
+    props: {
+      title,
+      description,
+      applyUrl,
+      tugOrganizationChartTitle,
+      tugOrganizationChartUrl,
+      howToBecomeTitle,
+      howToBecome,
+      progressTitle,
+      progressSummary,
+      progressStep1,
+      progressStep2,
+      peopleMemberTitle,
+      peopleMemberSummary,
+      peopleLeaderTitle,
+      peopleLeaderSummary,
+      tmc,
+      leader,
+    },
+  };
+};
+
+const People = ({
+  title,
+  description,
+  applyUrl,
+  tugOrganizationChartTitle,
+  tugOrganizationChartUrl,
+  howToBecomeTitle,
+  howToBecome,
+  progressTitle,
+  progressSummary,
+  progressStep1,
+  progressStep2,
+  peopleMemberTitle,
+  peopleMemberSummary,
+  peopleLeaderTitle,
+  peopleLeaderSummary,
+  tmc,
+  leader,
+}) => {
+  const howToBecomeDom = [];
+
+  howToBecome.forEach((item, index) => {
+    howToBecomeDom.push(<HowToBecome key={uuidV4()} {...item} />);
+    if (index !== howToBecome.length - 1) {
+      howToBecomeDom.push(
+        <div key={uuidV4()} className={styles.how_to_become_list_arrow}>
+          <img src="/images/people/arrow.svg" alt=">" />
+        </div>
+      );
+    }
+  });
```

Comment:
I feel a code smell because it's hard to guess at the first glance what's HTML structure it would generate.. Also, relying on mutable data is usually a bad design..

If I were you, I may implement the arrow with `:last-child` css or do it directly like:

```jsx
{howToBecome.map((item, idx) => <HowToBecome key={idx} {...item} />}
<div className={styles.how_to_become_list_arrow}>
  <img src="/images/people/arrow.svg" alt="next" />
</div>
```